Scope.Client/Loader/Features/Configs/ConfigsValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="ConfigsValidator.cs" company="Scope SL">
+// Copyright (c) Scope SL. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Scope.Client.Loader.Features.Configs
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using YamlDotNet.Core;
+    using YamlDotNet.Serialization;
+
+    /// <summary>
+    /// Configs validator.
```

Comment:
Make this more clear like, “A class used to validate the configs in ..config path..”